int8 whatAction
float32 howLong
---
bool isDone